tools/bpf2c/bpf2c.exe.manifest

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: MIT
+
+  Overrides the default heap implementation with segment heap.
+-->
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+  <assemblyIdentity name="microsoft.ebpf-for-windows.bpf2c" type="win32" version="0.9.0.0"></assemblyIdentity>
```

Comment:
If the 0.9.0 here is supposed to match the release version (meaning it should be updated each release), then you need to also update either `scripts\update-release-version.ps1` or `docs\ReleaseProcess.md`.  Check with @gtrevi to determine which.

Reply:
As per my understanding, yes, it is application binary version changes.

https://learn.microsoft.com/en-us/windows/win32/sbscs/assembly-versions

Reply:
I agree with @dthaler, `scripts\update-release-version.ps1` should be updated for automating the version within this file, similarly to how it's done for the `Product.wxs` file (i.e. using regex, see lines 20-24). In this case, being the AssemblyVersion's standard made of four digits, I'd just keep the last one to `.0`.

The `docs\ReleaseProcess.md` shouldn't instead need any update, as the script was intended to remove most of the manual steps for releasing eBPF.

Reply:
@Alan-Jowett : https://github.com/microsoft/ebpf-for-windows/blob/main/scripts/Set-Version.ps1 gives
$version = "$major_version.$minor_version.$revision_version" (0.9.0)
but the requirement for the manifest version field needs a **mandatory** 4-digit version (0.9.0.0)

Reply:
I would suggest leaving the last part as 0 then. I.e. assume major=x, minor=y, revision=z, then the version in the manifest would be x.y.z.0.

Reply:
Thanks @Alan-Jowett. As per the suggestion, made {version} to be populated in the bpf2c.exe.manifest by Set-Version.ps1 execution.